test/box-luatest/net_box_test.lua

```diff
@@ -258,3 +258,35 @@ g.test_connect_uri = function()
     t.assert_error_msg_content_equals(errmsg, net.connect, {}, '123')
     t.assert_error_msg_content_equals(errmsg, net.connect, 'localhost', true)
 end
+
+g.test_schemaless = function()
+    local c
+    -- fetch_schema = false
+    local schema_update_counter = 0
+    c = net.connect(g.server.net_box_uri, {fetch_schema = false})
+    t.assert_equals(c.state, 'active')
+    t.assert_equals(c.opts.fetch_schema, false)
+    t.assert_equals(c.space, nil)
+
+    c:on_schema_reload(function()
+        schema_update_counter = schema_update_counter + 1
+    end)
+    c:eval('box.schema.space.create("test_space")')
+    c:eval('box.space.test_space:drop()')
```

Comment:
Nit: IMO all potential artifacts created in a test case should be dropped in `after_test` callback so that if a test case fails for some reason, other test cases won't be affected. Let's drop the test spaces in the `after_test` callback, just in case:

```lua
g.after_test('test_schemaless', function()
    g.server:exec(function()
        if box.space.test1 then box.space.test1:drop() end
        if box.space.test2 then box.space.test2:drop() end
    end)
end
```

Reply:
Agree. Added after_test fixture.